config/application.yml.example

```diff
@@ -37,6 +37,9 @@ SMTP_PASSWORD: 'f00d'
 # MAILS_FROM: hello@example.com
 # MAIL_BCC: manager@example.com
 
+# Javascript error reporting via Bugsnag.
+#BUGSNAG_JS_KEY: ""
```

Comment:
this will need to come from ofn-install right?

Reply:
Yes. I'll do that today :+1: